applications/nrf5340_audio/README.rst

```diff
@@ -782,6 +782,12 @@ Selecting the CIS bidirectional communication
 The CIS unidirectional mode is the default operating mode for the application.
 You can switch to the bidirectional mode by adding the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
 
+Enabling the walkie talkie demo
+-------------------------------
+
+The walkie talkie demo is a bidirectional stream using the PDM microphone as input on each side.
+You can switch to the walkie talkie by adding the :kconfig:option:`CONFIG_WALKIE_TALKIE_DEMO` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
```

Comment:
```suggestion
Enabling the walkie-talkie demo
-------------------------------

The walkie-talkie demo is a bidirectional stream using the PDM microphone as input on each side.
You can switch to using the walkie-talkie by adding the :kconfig:option:`CONFIG_WALKIE_TALKIE_DEMO` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
```

- Does this Kconfig need to be added *in addition* to CONFIG_STREAM_BIDIRECTIONAL or *instead* of it?

Reply:
It will select CONFIG_STREAM_BIDIRECTIONAL, and all the other needed settings, so the user only needs to set that one Kconfig

Reply:
But it will also work if you set the STREAM_BIDIRECTIONAL first and then this, that's why I didn't specify further